pkg/operator/ceph/csi/spec.go

```diff
@@ -367,6 +367,44 @@ func startDrivers(namespace string, clientset kubernetes.Interface, ver *version
 			return errors.Wrap(err, "failed to load cephfs plugin service template")
 		}
 	}
+
+	pv, _ := clientset.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
+	drivers := []string{CephFSDriverName, RBDDriverName}
+	driversPV := false
+	for _, p := range pv.Items {
+		if p.Spec.CSI == nil {
+			continue
+		}
+		if p.Spec.CSI.VolumeAttributes["clusterID"] == tp.Namespace {
+			for _, d := range drivers {
+				if d == p.Spec.CSI.Driver {
+					driversPV = true
+				}
+			}
+
+		}
+	}
+	if !driversPV {
```

Comment:
This method is really long. I'd suggest moving the cleanup code to a new method that gets called from here.

Reply:
Done.